RationaleMCP/0031/grammar.md

```diff
@@ -355,12 +358,16 @@ end _F;
 
 ## Expressions
 
-> _expression_ → _simple-expression_ | _if-expression_
+> _decoration_ → **@** UNSIGNED-INTEGER
+
+> _expression_ → _expression-no-decoration_ _decoration_?
+
+> _expression-no-decoration_ → _simple-expression_ | _if-expression_
 
 > _if-expression_ →\
 > &emsp; **if** _expression_ **then** _expression_\
 > &emsp; ( **elseif** _expression_ **then** _expression_ )* \
-> &emsp; **else** _expression_
+> &emsp; **else** _expression-no-decoration_
```

Comment:
Web Meeting (Hans, Henrik, Gerd, Martin, Oliver)
A bit unfortunate to not having decorations on the else branch.
Using parenthesis could make it clear that the decoration belongs to the else branch.
```
Real x = if b then 1 @41 else 2 @42; // common use of decoration @42 belonging to entire if expr
```
```
Real x = if b then (1 @41) else (2 @42); // decoration @42 belonging to else branch, adding parenthesis also for @41
```
It's possible to omit the parenthesis in the other branches. 

Solution options:
A) as proposed, require as few parenthesis as possible.
B) require parenthesis for each branch and branching condition always. 1st example would be illegal.

Poll
A: 
B: Henrik, Gerd, Oliver
abstain: Hans, Martin

Decision:
Go for B.